spec/factories/books.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :book do
+    title "MyString"
```

Comment:
Is it ok if I address this during the project while writing tests? The current factories are the ones generated by default when I generate a model.